src/profiler.cpp

```diff
@@ -1667,6 +1667,12 @@ void Profiler::dumpOtlp(Writer& out, Arguments& args) {
     protobuf_mark_t scope_profiles_mark = otlp_buffer.startMessage(ResourceProfiles::scope_profiles);
     protobuf_mark_t profile_mark = otlp_buffer.startMessage(ScopeProfiles::profiles);
 
+    u64 time_nanos = (u64)_start_time * 1000000000ULL;
```

Comment:
I propose we change `_start_time` to contain nanoseconds (use `OS::nanotime`). Please track the usages of `_start_time` and convert to seconds as needed.

@apangin would you agree with this approach?